README.md

```diff
@@ -1,7 +1,37 @@
-## One-time setup
+# Heath Equity Tracker
+
+The main codebase for [Health Equity Tracker](https://healthequitytracker.org/).
+
+## Contributing
+To contribute to this project:
+1. Fork the repository on github.
```

Comment:
Maybe it would be helpful to link to this page here? https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo